jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,403 @@
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "from requests.auth import HTTPBasicAuth\n",
+    "import json\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth = HTTPBasicAuth(API_KEY, '')\n",
+    "session = requests.Session()\n",
+    "\n",
+    "# Authenticate\n",
+    "session.auth = (API_KEY, \"\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BASEMAP_API_URL = 'https://api.planet.com/basemaps/v1/mosaics'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "\n",
+    "basemap_params = {\n",
+    "    'name__contains': 'point_reyes_rolling_normalized_2020-04-30_mosaic'\n",
+    "}\n",
+    "\n",
+    "basemap_service_response = requests.get(url=BASEMAP_API_URL, params=basemap_params, auth=auth, headers=headers)\n",
+    "\n",
+    "basemaps = basemap_service_response.raise_for_status()\n",
+    "if basemap_service_response.status_code != 204:\n",
+    "    basemaps = json.loads(basemap_service_response.text)\n",
+    "print(basemaps)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosaic_ID = basemaps['mosaics'][0]['id']\n",
+    "mosaic_box = basemaps['mosaics'][0]['bbox']\n",
+    "mosaic_box_str = ','.join(map(str, mosaic_box))\n",
+    "\n",
+    "quad_params = {\n",
+    "    'mosaic_id': mosaic_ID,\n",
+    "    'bbox': mosaic_box_str,\n",
+    "}\n",
+    "\n",
+    "quads_url = \"{}/{}/quads\".format(BASEMAP_API_URL, mosaic_ID)\n",
+    "quadServiceResponse = requests.get(\n",
+    "    url=quads_url, params=quad_params, auth=auth, headers=headers)\n",
+    "quads = quadServiceResponse.json()\n",
+    "items = quads['items']\n",
+    "\n",
+    "quadIds = []\n",
+    "for i in items:\n",
+    "    quadId = i['id']\n",
+    "    quadIds.append(quadId)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create an order packet\n",
+    "---\n",
+    "Package up the details of your order in a JSON object and make a POST request, passing in the Orders URL, your JSON, your API key, and the content-type header. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "order_params = {\n",
+    "    \"name\": \"basemap order with quad_ids\",\n",
+    "    \"source_type\": \"basemaps\",\n",
+    "    \"products\": [\n",
+    "        {\n",
+    "            \"mosaic_name\": \"point_reyes_rolling_normalized_2020-04-30_mosaic\",\n",
+    "            \"quad_ids\": quadIds\n",
+    "        }\n",
+    "    ],\n",
+    "    \"tools\": [\n",
+    "        {\n",
+    "            \"reproject\": {\n",
+    "                \"projection\": \"EPSG:4326\",\n",
+    "                \"resolution\": 3.0 / 111000.0,\n",
```

Comment:
Same thing as your other notebook, explain above how you're getting to this resolution or why you're choosing it.